Tools/autotest/param_metadata/mdemit.py

```diff
@@ -5,9 +5,12 @@
 
 from param import known_param_fields
 from emit import Emit
+import time
 import re
 import os
 
+OFFSET = '' if os.getenv('DIFF') else '#'
```

Comment:
Can we add a comment explaining what this OFFSET is? and what it is used for

Reply:
Yep, will add in the morning :-)